tapdb/assets_store.go

```diff
@@ -218,6 +218,9 @@ type ActiveAssetsStore interface {
 	// disk.
 	UpsertAssetProof(ctx context.Context, arg ProofUpdate) error
 
+	FetchAssetID(ctx context.Context,
```

Comment:
nit: missing Godoc. Also, should this be called `QueryAssetPrimaryID`? To avoid confusion with the actual `AssetID` type (32-byte vs. numerical database ID). And query to signify we're querying by script key and/or outpoint?